pkg/sentry/fsimpl/fuse/connection.go

```diff
@@ -60,24 +74,260 @@ type FutureResponse struct {
+	ConnInit bool
+
+	// AsyncRead if read pages asynchronously.
+	// Only set in INIT.
+	AsyncRead bool
+
+	// AbortErr is true when FUSE will return an unique read error after abort.
+	// Only set in INIT.
+	AbortErr bool
+
+	// AtomicOTrunc is true when FUSE does not send a separate SETATTR request
+	// before open with O_TRUNC flag.
+	AtomicOTrunc bool
+
+	// ExportSupport is true if Filesystem supports NFS exporting.
+	// Only set in INIT.
+	ExportSupport bool
+
+	// WritebackCache is true if using write-back cache policy,
+	// false if using write-through policy.
+	WritebackCache bool
+
+	// ParallelDirops is true if allowing lookup and readdir in parallel,
+	// false if serialized.
+	ParallelDirops bool
+
+	// HandleKillpriv if fs handls killing suid/sgid/cap on write/chown/trunc.
+	HandleKillpriv bool
+
+	// CacheSymlinks if cache READLINK responses in page cache.
+	CacheSymlinks bool
+
+	/* Setting races on the following optimization-purpose flags are safe */
+
+	// NoOpen if open/release not implemented by fs.
+	NoOpen bool
+
+	// NoOpendir if opendir/releasedir not implemented by fs.
+	NoOpendir bool
+
+	// NoFsync if fsync not implemented by fs.
+	NoFsync bool
+
+	// NoFsyncdir if fsyncdir not implemented by fs.
+	NoFsyncdir bool
+
+	// NoFlush if flush not implemented by fs.
+	NoFlush bool
+
+	// NoSetxattr if setxattr not implemented by fs.
+	NoSetxattr bool
+
+	// NoGetxattr if getxattr not implemented by fs.
+	NoGetxattr bool
+
+	// NoListxattr if listxattr not implemented by fs.
+	NoListxattr bool
+
+	// NoRemovexattr if removexattr not implemented by fs.
+	NoRemovexattr bool
+
+	// NoLock if posix file locking primitives not implemented by fs.
+	NoLock bool
+
+	// NoAccess if access not implemented by fs.
+	NoAccess bool
+
+	// NoCreate if create not implemented by fs.
+	NoCreate bool
+
+	// NoInterrupt if interrupt not implemented by fs.
+	NoInterrupt bool
+
+	// NoBmap if bmap not implemented by fs.
+	NoBmap bool
+
+	// NoPoll if poll not implemented by fs.
+	NoPoll bool
+
+	// BigWrites if doing multi-page cached writes.
+	BigWrites bool
+
+	// DontMask don't apply umask to creation modes.
+	DontMask bool
+
+	// NoFLock if BSD file locking primitives not implemented by fs.
+	NoFLock bool
+
+	// NoFallocate if fallocate not implemented by fs.
+	NoFallocate bool
+
+	// NoRename2 if rename with flags not implemented by fs.
+	NoRename2 bool
+
+	// AutoInvalData use enhanced/automatic page cache invalidation.
+	AutoInvalData bool
+
+	// ExplicitInvalData Filesystem is fully reponsible for page cache invalidation.
+	ExplicitInvalData bool
+
+	// DoReaddirplus if the filesystem supports readdirplus.
+	DoReaddirplus bool
+
+	// ReaddirplusAuto if the filesystem wants adaptive readdirplus.
+	ReaddirplusAuto bool
+
+	// AsyncDio if the filesystem supports asynchronous direct-IO submission.
+	AsyncDio bool
+
+	// NoLseek if lseek() not implemented by fs.
+	NoLseek bool
+
+	// PosixAcl if the filesystem supports posix acls.
+	PosixAcl bool
```

Comment:
struct field `PosixAcl` should be `PosixACL` like `FileACLLo` in `pkg/sentry/fsimpl/ext/disklayout/inode_old.go`. Please see CodeReviewComments(https://github.com/golang/go/wiki/CodeReviewComments#initialisms])